docs/conf/portland/2023/news/announcing-cfp.rst

```diff
@@ -0,0 +1,46 @@
+:template: {{year}}/generic.html
+
+.. post:: Dec 01, 2022
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Announcing Call for Proposals
+=============================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`_.
+
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**For 2023 we are going back to an in-person event**.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
+
+The Call for Proposals will be open until **11:59 PM {{cfp.ends}} PST**.
```

Comment:
Do we want to already mention the high-level health and safety information? People might (will?) ask about this before they decide whether to submit a proposal

Reply:
@ericholscher wanna add some info here?